### ISSUE TITLE
Not showing up in Rails

### ISSUE THREAD

**Issue by adamw005** [michelson/dante2#50]
```
#Gemfile
gem 'dante2-editor'
gem 'react-rails' #, '~> 1.4.0'

#application.js
//= require jquery
//= require jquery_ujs
//= require react
//= require 'dante-vendors.js'
//= require 'Dante2.js'
//= require_tree .

#view
<div id="dante2-editor"></div>
<script type="text/javascript">
	new Dante({ el: "#dante2-editor"});
</script>
```

Console shows the Dante object being created but not showing. What am I missing?

**Comment by michelson** [comment]
are you importing the styles too ? check this https://github.com/michelson/dante2/wiki/Rails-styles-integration

**Comment by adamw005** [comment]
Yes, I'm using `@import "dante"` but the CSS files are not being loaded

**Comment by michelson** [comment]
ok @adamw005 , try removing the `#` to `el:` 👍

**Comment by adamw005** [comment]
Didn't seem to change anything

**Comment by Bodacious** [comment]
I think this is the same issue as #48

**Comment by michelson** [comment]
yup , try to use the `dante-vendors` branch.
I will release it in the next days

**Comment by michelson** [comment]
fixed in 0.5.0-rc1 install as Dante2@next
also check the new documentation in github page, use with webpacker
